src/stories/containers/Finances/components/DelegateExpenseTrend/HeaderDelegateExpense.stories.tsx

```diff
@@ -0,0 +1,249 @@
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import lightTheme from '@ses/styles/theme/light';
+import HeaderDelegateExpense from './HeaderDelegateExpense';
+import type { ComponentMeta } from '@storybook/react';
+
+export default {
+  title: 'Components/NewFinances/HeaderDelegateExpense',
+  component: HeaderDelegateExpense,
+  parameters: {
+    chromatic: {
+      viewports: [1194, 1280, 1440, 1920],
```

Comment:
you should pass this parameter per story as the different stories handle different viewports. eg: `HeaderDelegate1194` only works with 1194 breakpoint

Reply:
done